ocs_ci/ocs/ui/page_objects/object_buckets_tab.py

```diff
@@ -0,0 +1,67 @@
+import json
+
+import requests
+from selenium.webdriver.common.by import By
+
+from ocs_ci.ocs.ocp import get_ocp_url
+from ocs_ci.ocs.ui.mcg_ui import logger
+from ocs_ci.ocs.ui.page_objects.object_bucket_claims_tab import BucketsUI
+from ocs_ci.ocs.ui.page_objects.object_service import ObjectService
+
+
+class ObjectBucketsTab(ObjectService, BucketsUI):
+    def __init__(self):
+        super().__init__()
+
+    def delete_object_bucket_ui(self, delete_via, expect_fail, resource_name):
+        """
+        Delete an Object Bucket via the UI
+
+        delete_via (str): delete via 'OB/Actions' or via 'three dots'
```

Comment:
I think it would be nice to have arguments listed inside "Args:", like everywhere else in ocs_ci

Reply:
this is not a new code or docstring, I moved it to object_buckets_tab because UX was changed, all the code is the same as here https://github.com/red-hat-storage/ocs-ci/pull/8227/files/91344b6c0b7f740c9c17817bc021ddb4b5fd2dae#:~:text=via%20the%20UI-,delete_via%20(str)%3A%20delete%20via%20%27OB/Actions%27%20or%20via%20%27three%20dots%27,-expect_fail%20(str)%3A%20verify